### ISSUE TITLE
 reverseFromRelationship

### ISSUE THREAD

**Issue by sbc640964** [staudenmeir/eloquent-has-many-deep#163]
Feature cabin
Do not know if it belongs, but trying to ask
The function I am requesting is reverseFromRelationship
What I would like, is to get the corresponding function in the associated model and make it for the current model.

Example Fits your first sample in readme:
```
class Country extends Model
{
    use \Staudenmeir\EloquentHasManyDeep\HasRelationships;

    public function comments()
    {
        return $this->hasManyDeep(Comment::class, [User::class, Post::class]);
    }
}

class Comment extends Model
{
    use \Staudenmeir\EloquentHasManyDeep\HasRelationships;

    public function countries ()
    {
        return $this->hasReverseFromRelationship(Comment::class, 'comments');
    }
}
```

**Comment by sbc640964** [comment]
Thanks
It's nice of you to take every request seriously. :)

Successfully, Waiting for feature

**Comment by staudenmeir** [comment]
I released `v1.15.2` with reverse relationships:
https://github.com/staudenmeir/eloquent-has-many-deep#reversing-relationships
